src/case_gather/views.py

```diff
@@ -11,7 +11,13 @@
 @api_view(['GET', 'POST'])
 @permission_classes((IsAuthenticatedOrReadOnly,))
 def case_list(request):
+    '''
+    get:
+        Get a list of all cases.
 
+    post:
```

Comment:
I'm not sure if endpoints that haven't been implemented should be documented.

At least they should have a `NOT IMPLEMENTED` warning